content/zh/docs/devops-user-guide/how-to-integrate/sonarqube.md

```diff
@@ -143,12 +143,12 @@ weight: 11310
 
    ![Webhook 页面信息](/images/docs/zh-cn/devops-user-guide/tool-integration/integrate-sonarqube-into-pipelines/webhook-page-info.png)
 
-### 步骤 4：将 SonarQube 配置添加到 ks-installer
+### 步骤 4：将 SonarQube 配置添加到 ks-devops
 
-1. 执行以下命令编辑 `ks-installer`。
+1. 执行以下命令编辑 `devops-config`。
 
    ```bash
-   kubectl edit cc -n kubesphere-system ks-installer
+   kubectl edit cm -n kubesphere-devops-system devops-config
    ```
 
 2. 搜寻至 `devops`。添加字段 `sonarqube` 并在其下方指定 `externalSonarUrl` 和 `externalSonarToken`。
```

Comment:
Hi @LinuxSuRen , please don't forget to change this important step.

```yaml
sonarQube:
  host: http://10.77.1.201:31377 # The SonarQube IP address.
  token: 00ee4c512fc987d3ec3251fdd7493193cdd3b91d # The SonarQube admin token created above.

Reply:
Thanks for the reminder. See https://github.com/kubesphere/website/pull/2152/commits/aa4381a4b037e96bb37ee5834c1b16dea15203d1